<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #282828</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>471</width>
     <height>841</height>
    </rect>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabWidget{
                    border: none;
                    }
                </string>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <widget class="QWidget" name="vertexTab">
    <attribute name="title">
     <string>Vertex</string>
    </attribute>
    <widget class="QLabel" name="vertexLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(180,180,180);
                            font-size: 15px;
                            background-color: transparent;
                        </string>
     </property>
     <property name="text">
      <string>Vertex's Attributes:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="selectVertex">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox{
                            color: rgb(200, 200, 200);
                            border-style: none;
                            border-radius: 7px;
                            background-color: #383838;
                            padding-left: 13px;
                            }

                            QComboBox::drop-down
                            {
                            border: 0px; /* This seems to replace the whole arrow of the combo box */
                            }
                            QComboBox::down-arrow {
                            image: url(&quot;resource/gui/tool/down-arrow.png&quot;);
                            width: 9px;
                            height: 9px;
                            padding-right: 10px;
                            }
                        </string>
     </property>
    </widget>
    <widget class="QPushButton" name="applyVertexBtn">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>120</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
                            color: rgb(200, 200, 200);
                            border-style: 2px solid rgb(200, 200, 200);
                            border-radius: 7px;
                            background-color: #383838;
                            }

                            QPushButton:hover{
                            background-color: #303030;
                            }
                        </string>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Edge</string>
    </attribute>
    <widget class="QComboBox" name="selectEdge">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox{
                            color: rgb(200, 200, 200);
                            border-style: none;
                            border-radius: 7px;
                            background-color: #383838;
                            padding-left: 13px;
                            }

                            QComboBox::drop-down
                            {
                            border: 0px; /* This seems to replace the whole arrow of the combo box */
                            }
                            QComboBox::down-arrow {
                            image: url(&quot;resource/gui/tool/down-arrow.png&quot;);
                            width: 9px;
                            height: 9px;
                            padding-right: 10px;
                            }
                        </string>
     </property>
    </widget>
    <widget class="QLabel" name="edgeLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(180,180,180);
                            font-size: 15px;
                            background-color: transparent;
                        </string>
     </property>
     <property name="text">
      <string>Edge's Attributes:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="applyEdge">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>120</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
                            color: rgb(200, 200, 200);
                            border-style: 2px solid rgb(200, 200, 200);
                            border-radius: 7px;
                            background-color: #383838;
                            }

                            QPushButton:hover{
                            background-color: #303030;
                            }
                        </string>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutTab">
    <attribute name="title">
     <string>Layout</string>
    </attribute>
    <widget class="QComboBox" name="selectLayout">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#selectLayout{
                            color: rgb(200, 200, 200);
                            border-style: none;
                            border-radius: 7px;
                            background-color: #383838;
                            padding-left: 13px;
                            }

                            QComboBox::drop-down
                            {
                            border: 0px; /* This seems to replace the whole arrow of the combo box */
                            }
                            QComboBox::down-arrow {
                            image: url(&quot;resource/gui/tool/down-arrow.png&quot;);
                            width: 9px;
                            height: 9px;
                            padding-right: 10px;
                            }
                        </string>
     </property>
    </widget>
    <widget class="QLabel" name="layoutLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(180,180,180);
                            font-size: 15px;
                            background-color: transparent;
                        </string>
     </property>
     <property name="text">
      <string>Select Layout:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="selectLayoutEdgeWeight">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox{
                            color: rgb(200, 200, 200);
                            border-style: 2px solid rgb(200, 200, 200);
                            border-radius: 7px;
                            background-color: #383838;
                            padding-left: 13px;
                            padding-right: 10px;
                            }
                            QComboBox::drop-down
                            {
                            border: 0px; /* This seems to replace the whole arrow of the combo box */
                            }
                            QComboBox::down-arrow {
                            image: url(&quot;resource/gui/tool/down-arrow.png&quot;);
                            width: 9px;
                            height: 9px;
                            padding-right: 10px;
                            }
                        </string>
     </property>
    </widget>
    <widget class="QLabel" name="labelLayoutEdgeWeight">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(180,180,180);
                            font-size: 15px;
                            background-color: transparent;
                        </string>
     </property>
     <property name="text">
      <string>Select edge weight:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="applyLayoutBtn">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>190</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
                            color: rgb(200, 200, 200);
                            border-style: 2px solid rgb(200, 200, 200);
                            border-radius: 7px;
                            background-color: #383838;
                            }
                            QPushButton:hover{
                            background-color: #303030;
                            }
                        </string>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="clusterTab">
    <attribute name="title">
     <string>Cluster</string>
    </attribute>
    <widget class="QComboBox" name="selectClusteringAlgo">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#selectClusteringAlgo{
                            color: rgb(200, 200, 200);
                            border-style: 2px solid rgb(200, 200, 200);
                            border-radius: 7px;
                            background-color: #383838;
                            padding-left: 13px;
                            padding-right: 10px;
                            }
                            QComboBox::drop-down
                            {
                            border: 0px; /* This seems to replace the whole arrow of the combo box */
                            }
                            QComboBox::down-arrow {
                            image: url(&quot;resource/gui/tool/down-arrow.png&quot;);
                            width: 9px;
                            height: 9px;
                            padding-right: 10px;
                            }
                        </string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(180,180,180);
                            font-size: 15px;
                            background-color: transparent;
                        </string>
     </property>
     <property name="text">
      <string>Select Clustering Algorithm:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="selectClusteringAlgoEdgeWeight">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox{
                            color: rgb(200, 200, 200);
                            border-style: 2px solid rgb(200, 200, 200);
                            border-radius: 7px;
                            background-color: #383838;
                            padding-left: 13px;
                            padding-right: 10px;
                            }
                            QComboBox::drop-down
                            {
                            border: 0px; /* This seems to replace the whole arrow of the combo box */
                            }
                            QComboBox::down-arrow {
                            image: url(&quot;resource/gui/tool/down-arrow.png&quot;);
                            width: 9px;
                            height: 9px;
                            padding-right: 10px;
                            }
                        </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(180,180,180);
                            font-size: 15px;
                            background-color: transparent;
                        </string>
     </property>
     <property name="text">
      <string>Select edge weight:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="applyClusterBtn">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>190</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
                            color: rgb(200, 200, 200);
                            border-style: 2px solid rgb(200, 200, 200);
                            border-radius: 7px;
                            background-color: #383838;
                            }

                            QPushButton:hover{
                            background-color: #303030;
                            }
                        </string>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="filterTab">
    <attribute name="title">
     <string>Filter</string>
    </attribute>
    <widget class="QComboBox" name="selectFilterEdge">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox{
                            color: rgb(200, 200, 200);
                            border-style: none;
                            border-radius: 7px;
                            background-color: #383838;
                            padding-left: 13px;
                            }

                            QComboBox::drop-down
                            {
                            border: 0px; /* This seems to replace the whole arrow of the combo box */
                            }
                            QComboBox::down-arrow {
                            image: url(&quot;resource/gui/tool/down-arrow.png&quot;);
                            width: 9px;
                            height: 9px;
                            padding-right: 10px;
                            }
                        </string>
     </property>
    </widget>
    <widget class="QLabel" name="filterLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(180,180,180);
                            font-size: 15px;
                            background-color: transparent;
                        </string>
     </property>
     <property name="text">
      <string>Filter:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="applyFilterBtn">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>190</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
                            color: rgb(200, 200, 200);
                            border-style: 2px solid rgb(200, 200, 200);
                            border-radius: 7px;
                            background-color: #383838;
                            }
                            QPushButton:hover{
                            background-color: #303030;
                            }

                        </string>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="filterLeft">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>201</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
                            color: rgb(200, 200, 200);
                            border-style: 2px solid rgb(200, 200, 200);
                            border-radius: 7px;
                            background-color: #383838;
                            padding-left: 13px;
                            padding-right: 10px;
                            }

                        </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="filterRight">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>130</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
                            color: rgb(200, 200, 200);
                            border-style: 2px solid rgb(200, 200, 200);
                            border-radius: 7px;
                            background-color: #383838;
                            padding-left: 13px;
                            padding-right: 10px;
                            }

                        </string>
     </property>
    </widget>
    <widget class="QLabel" name="filterLabel_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>301</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(180,180,180);
                            font-size: 15px;
                            background-color: transparent;
                        </string>
     </property>
     <property name="text">
      <string>Min:                                           Max:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelFilterBtn">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>190</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
                            color: rgb(200, 200, 200);
                            border-style: 2px solid rgb(200, 200, 200);
                            border-radius: 7px;
                            background-color: #383838;
                            }
                            QPushButton:hover{
                            background-color: #303030;
                            }

                        </string>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="centralityTab">
    <attribute name="title">
     <string>Centrality</string>
    </attribute>
    <widget class="QComboBox" name="selectCentrality">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>381</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox{
                            color: rgb(200, 200, 200);
                            border-style: none;
                            border-radius: 7px;
                            background-color: #383838;
                            padding-left: 13px;
                            }

                            QComboBox::drop-down
                            {
                            border: 0px; /* This seems to replace the whole arrow of the combo box */
                            }
                            QComboBox::down-arrow {
                            image: url(&quot;resource/gui/tool/down-arrow.png&quot;);
                            width: 9px;
                            height: 9px;
                            padding-right: 10px;
                            }
                        </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(180,180,180);
                            font-size: 15px;
                            background-color: transparent;
                        </string>
     </property>
     <property name="text">
      <string>Centrality:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="applyCentralityBtn">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>180</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
                            color: rgb(200, 200, 200);
                            border-style: 2px solid rgb(200, 200, 200);
                            border-radius: 7px;
                            background-color: #383838;
                            }

                            QPushButton:hover{
                            background-color: #303030;
                            }
                        </string>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelCentralityBtn">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>180</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
                            color: rgb(200, 200, 200);
                            border-style: 2px solid rgb(200, 200, 200);
                            border-radius: 7px;
                            background-color: #383838;
                            }

                            QPushButton:hover{
                            background-color: #303030;
                            }
                        </string>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QComboBox" name="selectCentralityEdgeWeight">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>381</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox{
                            color: rgb(200, 200, 200);
                            border-style: none;
                            border-radius: 7px;
                            background-color: #383838;
                            padding-left: 13px;
                            }

                            QComboBox::drop-down
                            {
                            border: 0px; /* This seems to replace the whole arrow of the combo box */
                            }
                            QComboBox::down-arrow {
                            image: url(&quot;resource/gui/tool/down-arrow.png&quot;);
                            width: 9px;
                            height: 9px;
                            padding-right: 10px;
                            }
                        </string>
     </property>
    </widget>
    <widget class="QLabel" name="labelCentralityEdgeWeight">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>86</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(180,180,180);
                            font-size: 15px;
                            background-color: transparent;
                        </string>
     </property>
     <property name="text">
      <string>Edge weight:</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
